### ISSUE TITLE
Deploy storybooks to GH Pages

### ISSUE THREAD

**Issue by nicholasio** [ufersa/plataforma-sabia#201]
## Feature Description

Uma forma de fomentar o uso do storybooks seria fazer o deploy dele para o GH Pages por meio de github actions. Assim ficará bem fácil conferir o storybooks da master.

---------------

## Acceptance criteria

* 

## Implementation Brief

*

**Comment by nicholasio** [comment]
Uma boa referência:

- https://github.com/google/site-kit-wp/pull/1514
- https://github.com/google/site-kit-wp/issues/1459

**Comment by lcnogueira** [comment]
Sim. Apesar de não ser prioridade, é algo necessário. Podemos deixar para depois das features do lançamento 👍
